include/simgrid/plugins/energy.h

```diff
@@ -14,7 +14,7 @@ SG_BEGIN_DECL()
 XBT_PUBLIC void sg_host_energy_plugin_init();
 XBT_PUBLIC void sg_host_energy_update_all();
 XBT_PUBLIC double sg_host_get_consumed_energy(sg_host_t host);
-XBT_PUBLIC double sg_host_get_idle_consumption(sg_host_t host);
+XBT_PUBLIC double sg_host_get_wattidle_at(sg_host_t host, int pstate);
```

Comment:
I think that `sg_host_get_wattidle_at()` should exist in addition to `sg_host_get_idle_consumption()` since they don't really cover the same goal. `sg_host_get_idle_consumption()` is evaluating it at the current pstate.

In addition, if you want to remove a function from the public interface, you should deprecate the old one without removing it for one full year. 

Finally, `sg_host_get_wattidle_at()` should probably be named `sg_host_get_idle_consumption_at()` for sake of consistency.

Reply:
Actually `sg_host_get_idle_consumption()` calls `get_idle_consumption()` that simply retrieve the idle consumption of the first pstate [see here](https://github.com/simgrid/simgrid/blob/master/src/plugins/host_energy.cpp#L230),
that's why I wanted to replace it.

For the second point, I think it's better to keep consistency with the other functions `sg_host_get_wattmin_at()` and `sg_host_get_wattmax_at()` since they are getters of similar values.